service/src/main/java/org/broadinstitute/listener/relay/http/AvailabilityListener.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.listener.relay.http;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.availability.AvailabilityChangeEvent;
+import org.springframework.boot.availability.LivenessState;
+import org.springframework.boot.availability.ReadinessState;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Just a thought, but reading the article you linked in the PR description was very useful for me, I think it would be a good idea to link it in the code here as a comment as well https://www.baeldung.com/spring-liveness-readiness-probes

Reply:
Good idea! Will do.